core/federated/federate.c

```diff
@@ -88,8 +88,8 @@ federate_instance_t _fed = {.socket_TCP_RTI = -1,
                             .has_upstream = false,
                             .has_downstream = false,
                             .received_stop_request_from_rti = false,
-                            .last_sent_LTC = (tag_t){.time = NEVER, .microstep = 0u},
-                            .last_sent_NET = (tag_t){.time = NEVER, .microstep = 0u},
+                            .last_sent_LTC.time = NEVER,
+                            .last_sent_NET.time = NEVER,
```

Comment:
Nit: Why not set the microstep to 0, just to simplify things? It's probably fine but we might as well liberate ourselves from having to reason about UB from uninitialized memory.

Reply:
Yes I will do it. The reason for this change was that the original type of assignment was not allowed by our macOS compiler.

Reply:
OK, went back to NEVER_TAG since we don't have Werror on macOS for other reasons